phpunit-integration.xml.dist

```diff
@@ -1,26 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this file anymore? The test suites are all defined in `phpunit.xml.dist`.

Reply:
I think we can remove it. It's not referenced anywhere.